python/ml4ir/applications/ranking/tests/test_feature_layer.py

```diff
@@ -0,0 +1,166 @@
+from ml4ir.applications.ranking.tests.test_base import RankingTestBase
+from ml4ir.base.features.feature_fns import categorical as categorical_fns
+from ml4ir.base.features.feature_fns import sequence as sequence_fns
+from ml4ir.base.config.keys import SequenceExampleTypeKey
+
+import tensorflow as tf
+
+
+class RankingModelTest(RankingTestBase):
+    def test_bytes_sequence_to_encoding(self):
+        """
+        Goal:
+        Convert a string tensor to its corresponding sequence encoding
+        obtained through the bytes_sequence_to_encoding fn
```

Comment:
very minor suggestion: you can use "function (fn)" instead of "fn", + '.' at the end of the sentence.